src/page/Admin/AdminAccount.tsx

```diff
@@ -9,75 +9,72 @@ function AdminAccount() {
   const imgUploadInput = useRef<HTMLInputElement | null>(null);
 
   const onImageChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    if (event.target.files) {
-      setFile(event.target.files);
-
-      if (event.target.files[0]) {
-        const newFileURL = URL.createObjectURL(event.target.files[0]);
-        setFileURL(newFileURL);
-      } else {
-        setFileURL('');
-      }
-    }
-    if (imgUploadInput.current) {
-      imgUploadInput.current.click();
+    if (!event.target.files?.length) {
+      setFileURL('');
+      return;
     }
+
+    const newFileURL = URL.createObjectURL(event.target.files[0]);
+
+    setFile(event.target.files);
+    setFileURL(newFileURL);
+    imgUploadInput.current?.click();
   };
 
-  const onImageRemove = (): void => {
+  const removeImage = (): void => {
     URL.revokeObjectURL(fileURL);
     setFileURL('');
     setFile(null);
   };
 
-  const applyRegex = (url: string): string => {
+  const removeUsingRegex = (url: string): string => {
     const regex = new RegExp(/&?amount=\d+&?/g);
     return url.replace(regex, '');
   };
 
-  const submitHandler = async (event: React.MouseEvent<HTMLButtonElement>) => {
-    event.preventDefault();
-
-    if (file) {
-      const image = new Image();
-      const canvas = document.createElement('canvas');
-      const context = canvas.getContext('2d');
-
-      image.onload = () => {
-        canvas.width = image.width;
-        canvas.height = image.height;
-        if (context) {
-          context.drawImage(image, 0, 0, image.width, image.height);
-
-          const imageData = context.getImageData(0, 0, image.width, image.height);
-          const code = jsQR(imageData.data, imageData.width, imageData.height);
-
-          if (code) {
-            const decodedUrl: string = code.data;
-            const url = applyRegex(decodedUrl);
-            addAccount(url);
-          } else {
-            alert('QR코드가 인식되지 않았습니다.\n다시 업로드 바랍니다.');
-          }
-        }
-      };
-
-      image.src = fileURL;
-    } else {
+  const submitHandler = async () => {
+    if (!file) {
       alert('업로드할 이미지가 없습니다');
+      return;
     }
+
+    const image = new Image();
+    const canvas = document.createElement('canvas');
+    const context = canvas.getContext('2d');
+    image.src = fileURL;
```

Comment:
```suggestion
    const canvas = document.createElement('canvas');
    const context = canvas.getContext('2d');
    const image = new Image();
    image.src = fileURL;
```

선언과 사용부분은 최대한 붙여주시는게 좋습니다.